### ISSUE TITLE
v0.1.0 inconsistently causes framerate to drop to ~20fps and remain there until Nvidia Reflex is disabled

### ISSUE THREAD

**Issue by neilunger** [ishitatsuyuki/LatencyFleX#16]
While I am playing Overwatch and Apex Legends, there are times where I will be getting 100+ fps, and then the framerate will decide to drop to ~20fps and remain there until I disable Nvidia Reflex in-game.

For Apex, this only happens when I alt+tab back into the game while loading into a match (character select / champion screen). Even then, it seems inconsistent to reproduce -- so that isn't too much of an issue, as the few times it does happen, I can just quickly disable and re-enable reflex to fix it.

However with Overwatch, it usually happens multiple times per match, to where I have to disable reflex to play the game comfortably. I have noticed it pretty consistently happens whenever an enemy Sombra uses their ultimate. It is oddly inconsistent -- some matches it never happens, most matches it happens 3-5 times per game. It also usually happens the first time I load into a match. I tried moving the game off of my HDD and reinstalling on my SSD in case this was due to stutters from accessing the shader/DXVK cache, but that did not make a difference.

I saw that one of the dev branch build artifacts had a commit titled "Reduce jitter strength and pace down factor," which sounds like it could fix the issue (?), but the dev branch build artifacts are expired, so I am unable to download and test that. 

Are there any other reports of this happening? If the issue can't be reproduced, let me know which logs/additional info (if any) are needed. Thanks for the amazing work by the way, it at least makes a massive difference in Apex.

**Comment by ishitatsuyuki** [comment]
Yes, the drop on Alt-Tab with overwatch is a known issue. Overwatch does something wacky with the swapchain.

I haven't got any reports that it gets messed up in-match though, maybe you are having some extreme dips on those effects and it's triggering the failsafe which in turns resets state incorrectly.

As for Apex, I haven't got any reports of the same issue.

**Comment by laqyx** [comment]
I also get this issue in apex as described

**Comment by FuzzyQuills** [comment]
This also occurs with latest binaries, I just got this in Overwatch 2.

Regarding Sombra's ultimate being a consistent trigger; I've noticed Sombra's abilities sometimes trigger a huge pause as if the state cache for just her effects got invalidated. Same thing happens with Mercy's resurrect; I suspect the shader and PSO associated with the yellowish "glow" teammates have on rez does the same thing. Would be interesting to see if this happens on GPUs that support Graphics Pipeline Library, given the DXVK compilation stutter seems to be partially to blame. (The other part being what ishitatsuyuki mentioned before with OW doing funky things with the swapchain)
